docs/platforms/unreal/configuration/debug-symbols/index.mdx

```diff
@@ -21,6 +21,8 @@ You must configure which build configurations, target types, and platforms Sentr
 
 ![The Unreal Engine debug symbols upload configuration](./img/unreal-debug-symbols.png)
 
+Alternatively, the debug symbol upload can be enabled by setting the `SENTRY_UPLOAD_SYMBOLS_AUTOMATICALLY` environment variable to `True`. This can be especially helpful in CI/CD environments where manual configuration is impractical.
```

Comment:
```suggestion
Alternatively, debug symbol upload can be enabled by setting the `SENTRY_UPLOAD_SYMBOLS_AUTOMATICALLY` environment variable to `True`. This can be especially helpful in CI/CD environments where manual configuration is impractical.
```